src/CommunityToolkit.Maui/Converters/StringToListConverter.shared.cs

```diff
@@ -8,15 +7,41 @@ namespace CommunityToolkit.Maui.Converters;
 /// </summary>
 public class StringToListConverter : BaseConverterOneWay<string?, IEnumerable<string>>
 {
+	string separator = string.Empty;
+	IList<string> separators = Array.Empty<string>();
+
 	/// <summary>
-	/// The string that delimits the substrings in this string.
+	/// The string that delimits the substrings in this string
+	/// This value is superseded by the ConverterParameter, if provided
+	/// This value is also superseded by <see cref="Separators"/>, if provided
+	/// If ConverterParameter is null and <see cref="Separators"/> is not assigned, this property will be used to delimit the substrings
 	/// </summary>
-	public string Separator { get; set; } = string.Empty;
+	public string Separator
+	{
+		get => separator;
+		set => separator = value ?? throw new ArgumentNullException(nameof(value));
+	}
 
 	/// <summary>
-	/// The strings that delimits the substrings in this string.
+	/// The strings that delimits the substrings in this string
+	/// This value is superseded by the ConverterParameter, if provided
+	/// If ConverterParameter is null, this property will be used to delimit the substrings
 	/// </summary>
-	public IList<string> Separators { get; } = new List<string>();
+	public IList<string> Separators
+	{
+		get => separators;
+		set
+		{
+			ArgumentNullException.ThrowIfNull(value);
+
```

Comment:
Just curious if empty string is a valid separator

Reply:
Good point! I'll changes this verify against `string.IsNullOrEmpty` 👍

Reply:
Done ✅